src/views/domain-page/domain-page-failovers-filters/__tests__/domain-page-failovers-filters.test.tsx

```diff
@@ -0,0 +1,216 @@
+import { render, screen, userEvent } from '@/test-utils/rtl';
+
+import { type PageQueryParamValues } from '@/hooks/use-page-query-params/use-page-query-params.types';
+import { mockDomainPageQueryParamsValues } from '@/views/domain-page/__fixtures__/domain-page-query-params';
+import type domainPageQueryParamsConfig from '@/views/domain-page/config/domain-page-query-params.config';
+import { mockActiveActiveDomain } from '@/views/shared/active-active/__fixtures__/active-active-domain';
+import { type ActiveActiveDomain } from '@/views/shared/active-active/active-active.types';
+
+import { PRIMARY_CLUSTER_SCOPE } from '../../domain-page-failovers/domain-page-failovers.constants';
+import DomainPageFailoversFilters from '../domain-page-failovers-filters';
+
+describe(DomainPageFailoversFilters.name, () => {
+  it('renders both filter comboboxes and reset button', () => {
+    setup({});
+
+    expect(screen.getByText('Cluster Attribute Scope')).toBeInTheDocument();
+    expect(screen.getByText('Cluster Attribute Value')).toBeInTheDocument();
+    expect(screen.getByText('Reset filters')).toBeInTheDocument();
+  });
+
+  it('displays cluster attribute scope options including primary and domain scopes', async () => {
+    const { user } = setup({});
+
+    const comboboxes = screen.getAllByRole('combobox');
+    const scopeCombobox = comboboxes[0];
+    await user.click(scopeCombobox);
+
+    expect(screen.getByText(PRIMARY_CLUSTER_SCOPE)).toBeInTheDocument();
+    expect(screen.getByText('region')).toBeInTheDocument();
+  });
+
+  it('disables cluster attribute value combobox when scope is primary', () => {
+    setup({
+      queryParamsOverrides: {
+        clusterAttributeScope: PRIMARY_CLUSTER_SCOPE,
+      },
+    });
+
+    const inputs = screen.getAllByRole('textbox');
+    expect(inputs[1]).toBeDisabled();
+  });
+
+  it('enables cluster attribute value combobox when scope is not primary', () => {
+    setup({
+      queryParamsOverrides: {
+        clusterAttributeScope: 'region',
+      },
+    });
+
+    const inputs = screen.getAllByRole('textbox');
+    expect(inputs[1]).not.toBeDisabled();
```

Comment:
I wonder what inputs[1] refers to? 

Relying on index-1 may break if the input fields change in the future. Perhaps data-testid could help?